Misc/My Room/sp_greeting.csv

```diff
@@ -15,24 +15,24 @@ GOTOEXP_FAV1#0,"""Understood."""
 GOTOEXP_FAV0#0,"""I will go."""
 HELLO_REPORT_FAV4#0,"""マスター、お時間よろしければ<br>報告を聞いていただけませんか？"""
 HELLO_REPORT_FAV3#0,"""報告をしても<br>よろしいでしょうか？"""
-HELLO_REPORT_FAV2#0,"""戻りました。<br>報告があります"""
+HELLO_REPORT_FAV2#0,"""I've returned.<br>I have a report"""
 HELLO_REPORT_FAV1#0,"""There is a report to discuss"""
-HELLO_REPORT_FAV0#0,"""帰還しました"""
+HELLO_REPORT_FAV0#0,"""I've returned."""
 HELLO_OTHER#0,"""Welcome!"""
 GOODBYE_OTHER#0,"""Good-bye."""
 HELLO_FAV4#1,"""I was waiting!"""
-HELLO_FAV3#1,"""調子はどうだい？"""
+HELLO_FAV3#1,"""How are things going?"""
 HELLO_FAV2#1,"""Hey there."""
 HELLO_FAV1#1,"""HELLO_FAV1<br>Debgu_error"""
 HELLO_FAV0#1,"""HELLO_FAV0<br>Debgu_error"""
-GOODBYE_FAV4#1,"""名残惜しいね"""
-GOODBYE_FAV3#1,"""元気でね"""
+GOODBYE_FAV4#1,"""Too bad you have to go, right?"""
```

Comment:
Let's go with `It's time for you to go, isn't it?`

Reply:
I have a feeling that this may imply that the person saying this may not like the company of the one who leaves. For me, it sounds like "You stayed too long here, shoo!". Just my two cents though.

Reply:
I could see it as more sort of wistful regret, but that's with knowing what the context is already

Reply:
The support partners are very polite in their way of speech, so unless it was written as `It's time for you to leave` or `I think it's about time..`, you shouldn't see it as that type of tone.

Reply:
Then, "It's time for you to go, isn't it?" may do the trick? Or maybe "It's time for you to go, isn't it..?" may be better to show the reluctance in the "voice".

Reply:
The latter will do. As long as it works with english grammar.